scene/2d/remote_transform_2d.cpp

```diff
@@ -180,6 +180,10 @@ bool RemoteTransform2D::get_update_scale() const {
 	return update_remote_scale;
 }
 
+void RemoteTransform2D::force_update_cache() {
+    _update_cache();
```

Comment:
Style issue (uses spaces instead of tabs). Same in RemoteTransform. Otherwise it's good to merge :)